generator.go

```diff
@@ -0,0 +1,361 @@
+// Copyright 2017 The CRT Authors. All rights reserved.
```

Comment:
This code seems to be Windows-specific. Maybe it should be in file generator_windows.go. WDYT?